pkg/crc/machine/machine.go

```diff
@@ -680,23 +680,48 @@ func updateSSHKeyPair(sshRunner *crcssh.SSHRunner) error {
 	return err
 }
 
-func configureCluster(ocConfig oc.OcConfig, sshRunner *crcssh.SSHRunner, pullSecret string) (rerr error) {
+func configureCluster(ocConfig oc.OcConfig, sshRunner *crcssh.SSHRunner, baseDomainName, pullSecret, instanceIP string) (rerr error) {
 	sd := systemd.NewInstanceSystemdCommander(sshRunner)
 	if _, err := sd.Start("kubelet"); err != nil {
 		return fmt.Errorf("Error starting kubelet: %s", err)
 	}
 
+	proxy, err := network.NewProxyConfig()
+	if err != nil {
+		return err
+	}
+
```

Comment:
I'd somehow prefer to keep things separated, one helper function to configure pullsecret/cluster id, another one to configure proxy.

Reply:
@cfergeau sure, let me do that change